{
  "schema_name": "DoclingDocument",
  "version": "1.4.0",
  "name": "myfile",
  "origin": {
    "mimetype": "text/markdown",
    "binary_hash": 13447196763176184600,
    "filename": "myfile.md"
  },
  "furniture": {
    "self_ref": "#/furniture",
    "children": [],
    "content_layer": "furniture",
    "name": "_root_",
    "label": "unspecified"
  },
  "body": {
    "self_ref": "#/body",
    "children": [
      {
        "$ref": "#/texts/0"
      },
      {
        "$ref": "#/texts/1"
      },
      {
        "$ref": "#/texts/2"
      },
      {
        "$ref": "#/texts/3"
      },
      {
        "$ref": "#/texts/4"
      },
      {
        "$ref": "#/texts/5"
      },
      {
        "$ref": "#/texts/6"
      }
    ],
    "content_layer": "body",
    "name": "_root_",
    "label": "unspecified"
  },
  "groups": [],
  "texts": [
    {
      "self_ref": "#/texts/0",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "paragraph",
      "prov": [],
      "orig": "Forwarding X over ssh is normally fairly straightforward when you have the correct packages installed. I have  another post  about the errors that appear when you're missing the  xorg-x11-xauth  (CentOS, Fedora, RHEL) or  xauth  (Debian, Ubuntu) packages.",
      "text": "Forwarding X over ssh is normally fairly straightforward when you have the correct packages installed. I have  another post  about the errors that appear when you're missing the  xorg-x11-xauth  (CentOS, Fedora, RHEL) or  xauth  (Debian, Ubuntu) packages."
    },
    {
      "self_ref": "#/texts/1",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "paragraph",
      "prov": [],
      "orig": "Today's error was a bit different. Each time I accessed a particular Debian server via ssh with X forwarding requested, I saw this:",
      "text": "Today's error was a bit different. Each time I accessed a particular Debian server via ssh with X forwarding requested, I saw this:"
    },
    {
      "self_ref": "#/texts/2",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "code",
      "prov": [],
      "orig": "$ ssh -YC myserver.example.com\nX11 forwarding request failed on channel 0",
      "text": "$ ssh -YC myserver.example.com\nX11 forwarding request failed on channel 0",
      "captions": [],
      "references": [],
      "footnotes": [],
      "code_language": "unknown"
    },
    {
      "self_ref": "#/texts/3",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "paragraph",
      "prov": [],
      "orig": "The  xauth  package was installed and I found a  .Xauthority  file in root's home directory. Removing the  .Xauthority  file and reconnecting via ssh didn't help. After some searching, I stumbled upon a  GitHub gist  that had some suggestions for fixes.",
      "text": "The  xauth  package was installed and I found a  .Xauthority  file in root's home directory. Removing the  .Xauthority  file and reconnecting via ssh didn't help. After some searching, I stumbled upon a  GitHub gist  that had some suggestions for fixes."
    },
    {
      "self_ref": "#/texts/4",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "paragraph",
      "prov": [],
      "orig": "On this particular server, IPv6 was disabled. That caused the error. The quickest fix was to restrict sshd to IPv4 only by adding this line to  /etc/ssh/sshd_config :",
      "text": "On this particular server, IPv6 was disabled. That caused the error. The quickest fix was to restrict sshd to IPv4 only by adding this line to  /etc/ssh/sshd_config :"
    },
    {
      "self_ref": "#/texts/5",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "code",
      "prov": [],
      "orig": "AddressFamily inet",
      "text": "AddressFamily inet",
      "captions": [],
      "references": [],
      "footnotes": [],
      "code_language": "unknown"
    },
    {
      "self_ref": "#/texts/6",
      "parent": {
        "$ref": "#/body"
      },
      "children": [],
      "content_layer": "body",
      "label": "paragraph",
      "prov": [],
      "orig": "I restarted the ssh daemon and I was able to forward X applications over ssh once again.",
      "text": "I restarted the ssh daemon and I was able to forward X applications over ssh once again."
    }
  ],
  "pictures": [],
  "tables": [],
  "key_value_items": [],
  "form_items": [],
  "pages": {}
}